pkg/reconciler/kubernetes/tektonchain/tektonchain.go

```diff
@@ -46,6 +52,9 @@ const (
 	chainsContainerName = "tekton-chains-controller"
 	// Deployment Name
 	chainsDeploymentName = "tekton-chains-controller"
+
+	// secret installer set additional Annotation
+	SecretTISSigningAnnotation = "operator.tekton.dev/generated-signing-secret"
```

Comment:
if we do not use `SecretTISSigningAnnotation` outside package, we can make it `private`